src/client/connect/http.rs

```diff
@@ -216,51 +235,59 @@ impl<R: fmt::Debug> fmt::Debug for HttpConnector<R> {
     }
 }
 
-impl<R> Connect for HttpConnector<R>
+impl<R> tower_service::Service<Destination> for HttpConnector<R>
```

Comment:
we should probably support `TryFrom<Uri> for Destination` as well.

Reply:
Yeah, I think that isn't a horrible idea...I think we'd want to error out early though if they don't provide a scheme and authority?